### ISSUE TITLE
[Category] Short description

### ISSUE THREAD

**Issue by McJoe21** [sp614x/optifine#4777]
## Description of Issue
I was trying to play on aworld and it froze on the loading screen at 100%

## Steps to Reproduce
1. Launch minecraft with optifine
2. Click to play on a world
3. World loading gui

## OptiFine Version
1.16.2 HD U G3

## Installation Method
Standalone installer

## Log Files/Crash Reports
[crash-2020-09-04_21.05.57-client.txt](https://github.com/sp614x/optifine/files/5178536/crash-2020-09-04_21.05.57-client.txt)

## Prior Testing
If you've installed standalone:
- Does this happen in Vanilla without OptiFine? No

## Additional Information
Here goes any final remarks and everything else that might not fit in the previous sections.

**Comment by Rome6** [comment]
I have the same isue. I tried without optifine and the world works find.😔
Plz solution 🙏🙏

**Comment by McJoe21** [comment]
> I have the same isue. I tried without optifine and the world works find.😔
> Plz solution 🙏🙏

Did you read the log file?

**Comment by Rome6** [comment]
No, the game it's freezed all time and I have to force closing, I have
tried using control+C but it does not make nothing.
Currently I am using pre 4 for avoiding problems.
